src/config/navigation/defaultStackConfig.tsx

```diff
@@ -344,10 +346,17 @@ export const defaultStackConfig = ({
       screenComponent: VolunteerLoginScreen,
       screenOptions: { title: texts.screenTitles.volunteer.home }
     },
+    {
+      routeName: ScreenName.VolunteerLogout,
+      screenComponent: VolunteerLogoutScreen,
+      screenOptions: getScreenOptions({
+        noHeaderLeft: true,
+        cardStyleInterpolator: CardStyleInterpolators.forScaleFromCenterAndroid
+      })
```

Comment:
I understand that we want to use a different animation for navigation. This was built to use this animation, right?

Reply:
yes, this sets a different animation just for this route